h2/src/main/org/h2/table/Table.java

```diff
@@ -160,7 +161,6 @@ public abstract Index addIndex(Session session, String indexName,
      * @param key the primary key
      * @return the row
      */
-    @SuppressWarnings("unused")
     public Row getRow(Session session, long key) {
```

Comment:
this @SuppressWarnings annotations are useful here

Reply:
Reverted. For some reason Mars Eclipse doesn't like them, but I think you are right - they are useful.